scripts/build.bash

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-if [[ $PWD == */packages/arb-bridge-peripherals ]];
+if [[ $PWD == */token-bridge-contracts ]];
```

Comment:
this will break. the idea is that we want to know when the code is being used from inside `node_modules` to compile it only with the prod dependencies.
we could instead just stop doing this differentiation (but would need some other changes)

Reply:
pushed a commit for this